src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/GetBatchCertificateValidator.cs

```diff
@@ -26,48 +26,41 @@ public GetBatchCertificateRequestValidator(IStringLocalizer<GetBatchCertificateR
                 {
                     RuleFor(m => m).CustomAsync(async (m, context, cancellation) =>
                     {
-                        // NOTE: Currently we're making the Certificate & ILR record both mandatory
+                        // NOTE: Currently we're making the Certificate & ILR record both mandatory - this is wrong fixing it!
                         var requestedIlr = await ilrRepository.Get(m.Uln, m.StandardCode);
                         var sumbittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
+                        var existingCertificate = await certificateRepository.GetCertificate(m.Uln, m.StandardCode);
+                        var createdByCertificate = await certificateRepository.GetCertificateByOrgIdLastname(m.Uln, sumbittingEpao.EndPointAssessorOrganisationId, m.FamilyName);
 
-                        if (requestedIlr is null || !string.Equals(requestedIlr.FamilyName, m.FamilyName, StringComparison.InvariantCultureIgnoreCase))
-                        {
-                            context.AddFailure(new ValidationFailure("Uln", "Cannot find apprentice with the specified Uln, FamilyName & Standard"));
-                        }
-                        else if (sumbittingEpao is null)
+                        if (sumbittingEpao is null)
                         {
                             context.AddFailure(new ValidationFailure("UkPrn", "Specified UKPRN not found"));
                         }
                         else
                         {
                             var providedStandards = await standardService.GetEpaoRegisteredStandards(sumbittingEpao.EndPointAssessorOrganisationId);
 
-                            if (!providedStandards.Any(s => s.StandardCode == m.StandardCode))
+                            if (!providedStandards.Any(s => s.StandardCode == m.StandardCode) && createdByCertificate == null )
```

Comment:
What is the reasoning behind createdByCertificate being null along with not approved for the standard?

Reply:
What this is attempting to do - but please check my logic - is to allow certificates that were created by the EPAO, where that EPAO is no longer on the register for the standard. If the EPAO created the certificate then they can retrieve that data any time in the future.